src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -174,16 +180,20 @@ object DemuxFastqs {
       |the FASTQs, to assign each read to a sample.  Reads that do not match any sample within the given error tolerance
       |will be placed in the 'unmatched' file.
       |
-      |The output directory will contain one BAM file per sample in the sample sheet or metadata CSV file, plus a BAM for
-      |reads that could not be assigned to a sample given the criteria.  The output file names will be the concatenation
-      |of sample id, sample name, and sample barcode bases (expected not observed), delimited by `-`.  A metrics file
-      |will also be output providing analogous information to the metric described
+      |The type of output is specified with the `--output-type` option can be BAM (`--output-type BamOnly`), gzipped
+      |FASTQ (`--output-type FastqOnly`), or both (`--output-type Both`).
+      |
+      |For BAM output, the output directory will contain one BAM file per sample in the sample sheet or metadata CSV file,
+      |plus a BAM for reads that could not be assigned to a sample given the criteria.  The output file names will be the
+      |concatenation of sample id, sample name, and sample barcode bases (expected not observed), delimited by `-`.  A
+      |metrics file will also be output providing analogous information to the metric described
```

Comment:
I've never liked the link the Picard metric here.  Why not link to our own metric?